metrics/storage/webtooling.sh

```diff
@@ -51,6 +52,44 @@ Usage: $0 <count>
 EOF
 }
 
+clean() {
+	sudo systemctl restart containerd && sleep 1
```

Comment:
Do we need this `sleep`?

Reply:
Yes, I saw this issue in my environment.
However choosing a random number of seconds might not be enough.
Querying the service status might be better (every second for X times) might be better but also it might not be enough since the service might be active but trying to access containerd might result in an error since is not "ready".
Maybe a specialized restart_containerd function in lib that ensures containerd API is ready?

The easiest way to reproduce is to restart containerd and pull some image.

Reply:
thank you @jodh-intel, sleep removed